app/assets/sass/tai05102018.scss

```diff
@@ -77,7 +77,13 @@ hr {
 }
 
 .carry-over{
-  -ms-word-break: break-all;
+  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    display: inline-block;
+  }
+
+  @supports (-ms-accelerator:true) {
```

Comment:
I dont think you should be using this ? What was the use case?

https://developer.mozilla.org/en-US/docs/Web/CSS/-ms-accelerator